packages/calcite-components/src/components/navigation-logo/navigation-logo.tsx

```diff
@@ -96,12 +102,17 @@ export class CalciteNavigationLogo implements LoadableComponent {
   //
   // --------------------------------------------------------------------------
 
+  private renderIcon(): VNode {
+    return <calcite-icon class={CSS.icon} flipRtl={this.iconFlipRtl} icon={this.icon} scale="l" />;
```

Comment:
Can you add an explainer here covering why the [`getIconScale` util](https://github.com/Esri/calcite-design-system/blob/main/packages/calcite-components/src/utils/component.ts#L3) is not being used? cc @SkyeSeitz @ashetland

Reply:
Would Adam's description note serve as an explainer here?

"Because calcite-navigation and related components are offered in one scale, this unique icon case will use scale="l" due to the prominence and usage of the component, and to ensure icon adjacent to thumbnail are presented at equal sizes."

Reply:
Sure, maybe "Icon scale is not variable as the component does not have a `scale` property"? 

This is the case in Menu Item as well, and in Block, where we want to [add a similar `icon` prop](https://github.com/Esri/calcite-design-system/issues/4932), basically anywhere there is no `scale` on parent.

Reply:
![](https://media.tenor.com/pNhRfqCtyVEAAAAC/workaholics-i-like-that.gif)

Digging the explainers. ⛏️ Maybe for the code comment we can use the short version and we can update our conventions (or some other doc we can link to) with something along the lines of the description note?